### DISCUSSION TITLE
Soft integration in Operation settings (like in Everything in TC)

### DISCUSSION THREAD

**Discussion by LIMUNADE** [doublecmd/doublecmd#1113] [Suggestions]
### Description

I would like to suggest an option that already exists in Total Commander. 
![photo_2023-05-22_09-21-48](https://github.com/doublecmd/doublecmd/assets/20593218/0f0bc13b-0fb6-44b4-b8dd-0ff721fa671a)

So, if Everything app is launched, the size of all folders are showing straight up!

**Comment by LIMUNADE** [comment]
Search function is also applied 
![photo_2023-05-22_19-59-20](https://github.com/doublecmd/doublecmd/assets/20593218/1d804b96-a7d9-413a-be22-8bdb9ed07559)

**Comment by Skif-off** [comment]
DC has a [Everything](https://github.com/doublecmd/doublecmd/tree/master/plugins/dsx/everything/src) plugin (DSX plugin), but I don't know its current status.

**Reply by starfishpatkhoo** [reply]
> DC has a [Everything](https://github.com/doublecmd/doublecmd/tree/master/plugins/dsx/everything/src) plugin (DSX plugin), but I don't know its current status.

Yep, this is something I'm looking for. But there are no binaries for it on Windows... (build script does not build it...)

I can suggest that ordinarily Everything works with an IPC, but one can configure it to use `es.exe` to perform a search, which returns stuff in plain text .. For example, maybe a generic DLX plugin that allowed one to execute any command line with a `%s` or whatever as search term, then users can configure whatever search/indexing tool they want - locate, everything, grep, web, etc ... Maybe that is more cross-platform, and flexible at the same time...

**Comment by Dook1** [comment]
I support this proposal.

**Comment by bertil70** [comment]
This feature could convince me to switch from Total Commander to Double Commander (and maybe 2-3 other minor things like being able to filter with * as the first letter).

**Comment by LIMUNADE** [comment]
At it also could be helpful if folders could be sorted by size just as files could.

**Comment by Skif-off** [comment]
@LIMUNADE 

Go to "Configuration" in the main menu > Options... > [Files views](https://doublecmd.github.io/doc/en/configuration.html#ConfigView) > Sorting directories: try "sort like files and show first" or "sort like files".
Now Alt+Shift+Enter ([cm_CountDirContent](https://doublecmd.github.io/doc/en/cmds.html#cm_CountDirContent)) and try to change the sorting/sort direction.

**Reply by LIMUNADE** [reply]
Yeah, that did the trick. Thanks! But when I reenter the sub folders they lose size sorting, Is it the reality where I have to use Alt+Shift+Enter everytime when I want folders to be sorted by size?

**Reply by alexx2000** [reply]
https://doublecmd.h1n.ru/viewtopic.php?p=41847#p41847
